ocs_ci/framework/pytest_customization/ocscilib.py

```diff
@@ -264,6 +264,11 @@ def pytest_configure(config):
         # Add OCS related versions to the html report and remove
         # extraneous metadata
         markers_arg = config.getoption("-m")
+
+        # add logs url
+        if ocsci_config.RUN.get("logs_url"):
```

Comment:
are we forming logs_url somewhere in code or passing through yaml?

Reply:
Please document this value in:  https://github.com/red-hat-storage/ocs-ci/blob/master/conf/README.md

Also other request which @mbukatov will appreciate is to add this to xunit report. Can be done by other PR but please open issue to do that later on if not done in this PR.

Thanks!

Reply:
`logs_url` will is configured by Jenkins job (see [MR!468](https://gitlab.cee.redhat.com/ocs/ocs4-jenkins/-/merge_requests/468))